testdata/templates/bad-arg.yaml

```diff
@@ -0,0 +1,3 @@
+service: foo
+request:
+  name: ${user:foo
```

Comment:
Might be good to test requests with embedded non-string arguments, e.g., as in `yab -A peer-list:$(cat peers.yaml)`